code/platforms/win/app.py

```diff
@@ -1,11 +1,47 @@
 # defines the default app actions for windows
 
-from talon import Context, actions
+import win32gui
```

Comment:
`import win32gui` will cause an ImportError if we're not on windows, so you need to try/except fence this.

Reply:
Okay, used 'if' instead of 'try' because we don't want to ignore import errors when we are on windows (and so we would have needed an 'if' anyways).